.ci/pytorch/test.sh

```diff
@@ -105,14 +105,6 @@ if [[ "$TEST_CONFIG" == *crossref* ]]; then
   export PYTORCH_TEST_WITH_CROSSREF=1
 fi
 
-if [[ "$TEST_CONFIG" == *dynamo* ]]; then
-  export PYTORCH_TEST_WITH_DYNAMO=1
-fi
-
-if [[ "$TEST_CONFIG" == *inductor* ]]; then
-  export PYTORCH_TEST_WITH_INDUCTOR=1
-fi
```

Comment:
I feel like it doesn't make much sense for this to be a temporary state. You haven't removed the dynamo/inductor CI jobs, so we're effectively running the tests twice under the same config. If you're going the gradual enablement route, you should work out how to NOT run tests under these shards unless they've been explicitly enabled for dynamo execution.